### PR TITLE
Feature/npc dialogue

### PR DESCRIPTION
**Pull request by silfaber**
still fixing this up, also fixing that merge error

### CODE REVIEW

**Comment by silfaber** [comment]
fucking CurrentMaximizeLayout.dwlt merge error

update: fixed it

**Comment by silfaber** [comment]
i know how to set it as draft

but how do I set it as not draft

**Comment by silfaber** [comment]
gonna add the mobile dialogue controls on this branch first as well

File: Assets/Scripts/Dialogue/InkManager.cs [hunk 1]
```diff
@@ -17,11 +17,9 @@ public class InkManager : MonoBehaviour
     private Transform choiceButtonParent;
     [SerializeField]
     private Image playerImage, npcImage;
-    [SerializeField]
-    private List<CharacterSpriteSO> characterSprites = new List<CharacterSpriteSO>();
 
     private Story story;
-    private bool isDialogueActive = false;
+    public bool isDialogueActive = false;
```
Comment (JdeHaan2001) [inline Assets/Scripts/Dialogue/InkManager.cs]: Make sure to use pascal case for public variables

File: Assets/Scripts/Dialogue/InkManager.cs [hunk 2]
```diff
@@ -110,6 +100,8 @@ private void InstantiateChoiceButtons()
         }
     }
 
+    //Tags are currently unused, I am planning to use them for changing the portraits depending on
```
Comment (JdeHaan2001) [inline Assets/Scripts/Dialogue/InkManager.cs]: Can you use the ///summary when commenting functions

**Review by JdeHaan2001 — APPROVED** [review]
I approved the pull request, but please do first change the things I noted

**Pull request merged**